ui/client/common/ProcessStateUtils.js

```diff
@@ -1,39 +1,124 @@
 import _ from 'lodash'
 
+import InlinedSvgs from "../assets/icons/InlinedSvgs"
+
 export default {
-  ACTION_DEPLOY: "DEPLOY",
-  ACTION_CANCEL: "CANCEL",
+  ACTIONS: {
+    NOT_DEPLOYED: null,
+    DEPLOY: "DEPLOY",
+    CANCEL: "CANCEL",
+  },
+
+  STATUSES: {
+    UNKNOWN: "UNKNOWN",
+    NOT_DEPLOYED: "NOT_DEPLOYED",
+    DURING_DEPLOY: "DURING_DEPLOY",
+    DEPLOYED: "DEPLOYED",
+    RUNNING: "RUNNING",
+    CANCELED: "CANCELED",
+    RESTARTING: "RESTARTING",
+    FAILED: "FAILED",
+    ERROR: "ERROR",
+    FINISHED: "FINISHED",
+  },
+
+  STATUS_TOOLTIPS: {
```

Comment:
As I understand, these are tooltips that will be used if api won't return own tooltip? If it's true, maybe we can explicitly name? Like `DEFAULT_STATUS_TOOLTIPS`? Don't you think that it will be confusing that part tooltips preparing will be on BE side, and other part on FE side?

Reply:
I agree with taht we can named it DEFAULT_*. It can be confusing, but I have done it in this way because of optimization case - all tooltips and icons now are cached by http server.